plugins/modules/ovirt_cluster.py

```diff
@@ -179,6 +179,14 @@
             - "C(false) - Override the global setting to I(false)."
             - "C(inherit) - Use value which is set globally."
         choices: ['true', 'false', 'inherit']
+    migration_encrypted:
+        description:
+            - "If I(True) encryption is used during live migration of the virtual machine."
+            - "Following options are supported:"
+            - "C(true) - Override the global setting to I(true)."
```

Comment:
Shouldn't we use `True` instead of `true`?

Reply:
The smaller case is used in `migration_compressed` and `migration_auto_converge` so it is better to keep it same everywhere.

Reply:
Shouldn't we alway use the values True/False which are compatible with Python boolean values?

Reply:
The type of the `migration_encrypted` is a string and the ovirt sdk can take the value with first letter small or large case, so it would not affect anything. 
I want to keep it as it is because it is same implementation as in `migration_compressed` and `migration_auto_converge`. 
If you think we should update it to large I'll update it everywhere.

Reply:
OK